### PR TITLE
OSDOCS3023: Updating clusters overview

### PR DESCRIPTION
**Pull request by xenolinux**
OCP version: 4.8 only
[Preview](https://deploy-preview-43645--osdocs.netlify.app/openshift-enterprise/latest/updating/index.html)
[Jira](https://issues.redhat.com/browse/OSDOCS-3023)
QE contact @jiajliu

### CODE REVIEW

File: updating/index.adoc [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+[id="updating-clusters-overview"]
+= Updating clusters overview
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+[id='updating-clusters-overview-understand-update-service']
```
Comment (jiajliu) [inline updating/index.adoc:11]: In v4.8 [doc](https://docs.openshift.com/container-platform/4.8/updating/understanding-openshift-updates.html), there is not `Understanding the OpenShift Update Service`.

File: updating/index.adoc [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+== Understanding the OpenShift Update Service
+xref:../updating/updating-restricted-network-cluster.adoc#update-service-overview_updating-restricted-network-cluster[About the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. 
+
+['updating-clusters-overview-install-and-configure-update-service']
+== Installing and configuring the OpenShift Update Service
```
Comment (jiajliu) [inline updating/index.adoc:16]: This part was moved into `Updating a restricted network cluster` in [v4.8](https://docs.openshift.com/container-platform/4.8/updating/updating-restricted-network-cluster.html).

File: updating/index.adoc [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+* xref:../updating/installing-update-service.adoc#update-service-create-service[Creating an OpenShift Update Service application]
+* xref:../updating/installing-update-service.adoc#update-service-uninstall[Deleting an OpenShift Update Service application]
+
+['updating-clusters-overview-upgrade-channels-and-releases']
+== Upgrading channels and releases
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrading channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are connected to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. If you do not want the Cluster Version Operator to fetch available updates from the {product-title} Update Service (OSUS), you can use the `oc adm upgrade channel` command in the `oc` to configure an empty channel. For more information, check the following information:
+
```
Comment (jiajliu) [inline updating/index.adoc:27]: In v4.8, we don't support `oc adm upgrade channel`. Refer to https://docs.openshift.com/container-platform/4.8/updating/understanding-upgrade-channels-release.html

File: updating/index.adoc [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+['updating-clusters-overview-update-cluster-using-web-console']
+== Updating a cluster within a minor version using the web console
+xref:../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster within a minor version using the web console]: You can update an {product-title} cluster by using the web console. You can the update a cluster between minor versions. For more information, check the following items:
+
+* xref:../updating/updating-cluster-within-minor.adoc#update-upgrading-web_updating-cluster-within-minor[Updating a cluster by using the web console]
+* xref:../updating/updating-cluster-within-minor.adoc#update-changing-update-server-web_updating-cluster-within-minor[Changing the update server by using the web console]
+
+['updating-clusters-overview-update-cluster-using-cli']
+== Updating a cluster within a minor version using the command-line interface (CLI)
```
Comment (jiajliu) [inline updating/index.adoc:48]: Following sections are not align with https://docs.openshift.com/container-platform/4.8/updating/updating-cluster-cli.html

File: updating/index.adoc [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-cluster[Updating a restricted network cluster]: A restricted network environment is one in which your cluster nodes cannot access the internet. You can update a restricted network {product-title} cluster by using the `oc`. For more information, check the following items:
+
+* xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-mirror-host[Preparing your mirror host]
+* xref:../updating/updating-restricted-network-cluster.adoc#installation-adding-registry-pull-secret_updating-restricted-network-cluster[Configuring credentials that allow images to be mirrored]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-mirror-repository_updating-restricted-network-cluster[Mirroring the {product-title} image repository]
+* xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-image-signature-configmap[Creating the image signature config map]
+* xref:../updating/updating-restricted-network-cluster.adoc#machine-health-checks-pausing_updating-restricted-network-cluster[Pausing a MachineHealthCheck resource]
```
Comment (jiajliu) [inline updating/index.adoc:82]: No this step in v4.8 https://docs.openshift.com/container-platform/4.8/updating/updating-restricted-network-cluster.html.

File: updating/index.adoc (hunk 1 above)
Comment (xenolinux) [reply updating/index.adoc]: Addressed

File: updating/index.adoc (hunk 2 above)
Comment (xenolinux) [reply updating/index.adoc]: Addressed- Removed this individual section and added it under `Updating a restricted network cluster`

File: updating/index.adoc (hunk 4 above)
Comment (xenolinux) [reply updating/index.adoc:39]: Addressed

File: updating/index.adoc (hunk 5 above)
Comment (xenolinux) [reply updating/index.adoc]: Addressed

File: updating/index.adoc [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+[id='updating-clusters-overview-understanding-container-platform-updates']
+== Understanding OpenShift Container Platform updates
+xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[About the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs.
```
Comment (jiajliu) [inline updating/index.adoc:13]: > ../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[About the OpenShift Update Service]
It seems duplicated with `../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates]`

File: updating/index.adoc (hunk 3 above)
Comment (jiajliu) [reply updating/index.adoc:18]: ack

File: updating/index.adoc (hunk 4 above)
Comment (jiajliu) [reply updating/index.adoc:39]: ack

File: updating/index.adoc [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+* xref:../updating/updating-restricted-network-cluster.adoc#installation-adding-registry-pull-secret_updating-restricted-network-cluster[Configuring credentials that allow images to be mirrored]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-mirror-repository_updating-restricted-network-cluster[Mirroring the {product-title} image repository]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-restricted_updating-restricted-network-cluster[Updating the restricted network cluster]
+* xref:../updating/updating-restricted-network-cluster.adoc#images-configuration-registry-mirror_updating-restricted-network-cluster[Configuring image registry repository mirroring]
+* xref:../updating/updating-restricted-network-cluster.adoc#generating-icsp-object-scoped-to-a-registry_updating-restricted-network-cluster[Widening the scope of the mirror image catalog to reduce the frequency of cluster node reboots]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-service-install[Installing the OpenShift Update Service Operator]
```
Comment (jiajliu) [inline updating/index.adoc:72]: There are two parts in [this page](https://docs.openshift.com/container-platform/4.8/updating/updating-restricted-network-cluster.html):
1. Updating a restricted network cluster using the CLI
2. Updating a restricted network cluster using the OpenShift Update Service

We seems missing the summary/brief description about part 2 at the [beginning](https://github.com/openshift/openshift-docs/blob/a9735554fc6c29e6502ff823a171074572e29f3a/updating/index.adoc#updating-a-restricted-network-cluster), and only links here will mislead users. I suggest we should also add a brif description before the detail steps as following:
```
[Installing the OpenShift Update Service Operator](https://github.com/openshift/openshift-docs/blob/a9735554fc6c29e6502ff823a171074572e29f3a/updating/updating-restricted-network-cluster.adoc#update-service-install)

[Creating an OpenShift Update Service application](https://github.com/openshift/openshift-docs/blob/a9735554fc6c29e6502ff823a171074572e29f3a/updating/updating-restricted-network-cluster.adoc#update-service-create-service)

[Deleting an OpenShift Update Service application](https://github.com/openshift/openshift-docs/blob/a9735554fc6c29e6502ff823a171074572e29f3a/updating/updating-restricted-network-cluster.adoc#update-service-delete-service)

[Uninstalling the OpenShift Update Service Operator](https://github.com/openshift/openshift-docs/blob/a9735554fc6c29e6502ff823a171074572e29f3a/updating/updating-restricted-network-cluster.adoc#update-service-uninstall)
```

File: updating/index.adoc (hunk 1 above)
Comment (jiajliu) [reply updating/index.adoc]: ack

File: updating/index.adoc (hunk 2 above)
Comment (jiajliu) [reply updating/index.adoc]: ack

File: updating/index.adoc (hunk 6 above)
Comment (xenolinux) [reply updating/index.adoc:13]: Addressed

File: updating/index.adoc (hunk 7 above)
Comment (xenolinux) [reply updating/index.adoc:72]: Addressed- Added the sentence which implies that customers can update using the Openshift update service.

**Comment by xenolinux** [comment]
@jiajliu I addressed all the comments, does this look good now?

File: updating/index.adoc (hunk 6 above)
Comment (jiajliu) [reply updating/index.adoc:13]: ack

File: updating/index.adoc (hunk 7 above)
Comment (jiajliu) [reply updating/index.adoc:72]: ack

File: updating/index.adoc (hunk 5 above)
Comment (jiajliu) [reply updating/index.adoc]: ack

**Comment by jiajliu** [comment]
lgtm

File: updating/index.adoc [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-mcp_update-using-custom-machine-config-pools[Creating machine config pools to perform a canary rollout update]
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-pause_update-using-custom-machine-config-pools[Pausing the machine config pools]
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-update_update-using-custom-machine-config-pools[Performing the cluster update]
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-unpause_update-using-custom-machine-config-pools[Unpausing the machine config pools]
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-mcp-remove_update-using-custom-machine-config-pools[Moving a node to the original machine config pool]
```
Comment (nalhadef) [inline updating/index.adoc:53]: Change all 4 instances of "config" in the link text to "configuration."

File: updating/index.adoc [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+
+* xref:../updating/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute-hooks[Optional: Adding hooks to perform Ansible tasks on RHEL machines]
+* xref:../updating/updating-cluster-rhel-compute.adoc#rhel-compute-updating-minor_updating-cluster-rhel-compute[Updating RHEL compute machines in your cluster]
+
+['updating-clusters-overview-update-restricted-network-cluster']
+== Updating a restricted network cluster
+xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-cluster[Updating a restricted network cluster]: A restricted network environment is one in which your cluster nodes cannot access the internet. You can update a restricted network {product-title} cluster by using the `oc`. You can also update a restricted network {product-title} cluster by using the OpenShift Update Service. For more information, see the following items:
```
Comment (nalhadef) [inline updating/index.adoc:64]: Delete "items" from the end of Line 64.

**Review by nalhadef — COMMENTED** [review]
A few changes to incorporate. Thanks!

File: updating/index.adoc [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+['updating-clusters-overview-upgrade-channels-and-releases']
+== Understanding upgrading channels and releases
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrading channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are connected to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. For more information, see the following information:
+
+* xref:../updating/understanding-upgrade-channels-release.adoc#upgrade-version-paths_understanding-upgrade-channels-releases[Upgrading version paths]
+* xref:../updating/understanding-upgrade-channels-release.adoc#fast-stable-channel-strategies_understanding-upgrade-channels-releases[Understanding Fast and stable channel use and strategies]
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply updating/index.adoc:20]: Addressed

File: updating/index.adoc [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+== Preparing to perform an EUS-to-EUS update
+xref:../updating/preparing-eus-eus-upgrade.adoc#preparing-eus-eus-upgrade[Preparing to perform an EUS-to-EUS update]: Due to fundamental Kubernetes design, all {product-title} updates between minor versions must be serialized. You must update from {product-title} 4.8 to 4.9, and then to 4.10. You cannot update from {product-title} 4.8 to 4.10 directly. However, if you want to update between two Extended Update Support (EUS) versions, you can do so by incurring only a single reboot of non-master hosts. For more information, see the following item:
+
+* xref:../updating/preparing-eus-eus-upgrade.adoc#updating-eus-to-eus-upgrade_eus-to-eus-upgrade[Updating EUS-to-EUS]
+
+['updating-clusters-overview-update-cluster-using-web-console']
+== Updating a cluster using the web console
+xref:../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster within a minor version using the web console]: You can update an {product-title} cluster by using the web console. You can the update a cluster between minor versions. For more information, see the following items:
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply updating/index.adoc:32]: Addressed

File: updating/index.adoc [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+
+* xref:../updating/updating-cluster-within-minor.adoc#update-using-custom-machine-config-pools-canary_updating-cluster-within-minor[Performing a canary rollout update]
+* xref:../updating/updating-cluster-within-minor.adoc#update-upgrading-web_updating-cluster-within-minor[Updating a cluster by using the web console]
+* xref:../updating/updating-cluster-within-minor.adoc#update-changing-update-server-web_updating-cluster-within-minor[Changing the update server by using the web console]
+
+['updating-clusters-overview-update-cluster-using-cli']
+== Updating a cluster within a minor version using the command-line interface (CLI)
+xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version using the CLI]: You can update an {product-title} cluster by using the `oc`. You can the update a cluster between minor versions. For more information, see the following actions:
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply updating/index.adoc:40]: Addressed

File: updating/index.adoc (hunk 9 above)
Comment (xenolinux) [reply updating/index.adoc:64]: Addressed

**Pull request merged**